packages/@react-stately/layout/src/ListLayout.ts

```diff
@@ -373,30 +381,36 @@ export class ListLayout<T> extends Layout<Node<T>> implements KeyboardDelegate {
   }
 
   getKeyPageAbove(key: Key) {
-    let layoutInfo = this.getLayoutInfo(key);
-    let pageY = Math.max(0, layoutInfo.rect.y + layoutInfo.rect.height - this.virtualizer.visibleRect.height);
-    while (layoutInfo && layoutInfo.rect.y > pageY && layoutInfo) {
-      let keyAbove = this.getKeyAbove(layoutInfo.key);
-      layoutInfo = this.getLayoutInfo(keyAbove);
-    }
+    let layoutInfo = this.getLayoutInfo(key || this.getFirstKey());
```

Comment:
Should we account for null keys here as well? What was the reason for the change to add `getFirstKey` here?

Reply:
Ah yeah, def should account for null keys and I think we can actually remove the `getFirstKey` here since this is the page up case. That would mean this can go back to being
```suggestion
    let layoutInfo = this.getLayoutInfo(key);
```
For `getKeyPageBelow` the `getFirstKey` is there to provide a starting point in case the user opened the menu by clicking the menu trigger -> no item is auto focused -> user presses page down. That one should account for a null key with something like this I guess:
```let layoutInfo = this.getLayoutInfo(key != null ? key : this.getFirstKey());```

Reply:
I'll push something up for this